packages/server/test/course.integration.ts

```diff
@@ -251,5 +251,56 @@ describe('Course Integration', () => {
 
       expect(events.length).toBe(0);
     });
+
```

Comment:
Could we put these tests in their own describe block like how it's done above?
`describe('DELETE, /courses/:id/ta_location/:room', () => {`